packages/injected-script/package.json

```diff
@@ -12,5 +12,8 @@
   "scripts": {
     "start": "rollup -c -w",
     "build": "rollup -c"
+  },
+  "dependencies": {
+    "web3-core-helpers": "1.5.2"
```

Comment:
Please do not add dependencies to this package.

Reply:
Move to the root package?

Reply:
No. Keep the related type any is ok